src/constants/layers-urls.js

```diff
@@ -201,9 +201,9 @@ export const LAYERS_URLS = {
 
   // Protected areas WDPA
   [PROTECTED_AREAS_VECTOR_TILE_LAYER]:
-    'https://tiles.arcgis.com/tiles/IkktFdUAcY3WrH25/arcgis/rest/services/ProtectedAreas_202504/VectorTileServer',
+    'https://tiles.arcgis.com/tiles/IkktFdUAcY3WrH25/arcgis/rest/services/ProtectedAreas_20250416/VectorTileServer',
```

Comment:
Im not seeing the protected areas anymore in the map. Is it possible to test the changes on staging before doing this PR to production?

Reply:
Protected areas are showing up fine here - https://half-earth-v3.vercel.app/dataGlobe. I noticed sometimes it doesn't show but then I refresh and it shows fine

Reply:
Ok, I can see them after reloading. Maybe we will have to check why they don't show at first.

Another issue: This might be on purpose but the layers are cleared when I reload the page. Maybe this was required at some point